chapters/04-function-development.Rmd

```diff
@@ -8,35 +8,115 @@
 - Mostly *how workflow actually looks like*.
 - Building functions up slowly, making small targeted functions that build up
   into a bigger more complex functions
-- Process control (if-else, stop, return, switch)
+- Process control (if, stop)
 - Dependency management
+
+TODO: should this be here or later?
```

Comment:
Doesn't dependency management fit betterwith packages?

Reply:
Reading further down it seems like this is more about using functions from packages, I don't know if I would call that dependencies, I think the longer description is better. It probably goes better here since it it moslty aabout functions

Reply:
Yea, good point.